src/AllTravelers.js

```diff
@@ -10,46 +10,51 @@ class AllTravelers {
   getTrips(theStatus) {
     if (theStatus === 'pending') {
       return this.trips.filter(trip => trip.date >= moment().format('YYYY/MM/DD') && trip.status === 'pending').map(trip => {
-        const tripsObj = {};
-        tripsObj.name = this.travelers.find(traveler => traveler.id === trip.userID).name;
-        tripsObj.date = trip.date;
-        tripsObj.duration = trip.duration;
-        tripsObj.travelers = trip.travelers;
-        tripsObj.destination = this.destinations.find(dest => dest.id === trip.destinationID).destination;
-        tripsObj.status = trip.status;
-        return tripsObj;
+        const tripsDetails = {};
+        tripsDetails.name = this.travelers.find(traveler => traveler.id === trip.userID).name;
+        tripsDetails.date = trip.date;
+        tripsDetails.duration = trip.duration;
+        tripsDetails.travelers = trip.travelers;
+        tripsDetails.destination = _.find(this.destinations, dest => dest.id === trip.destinationID).destination;
```

Comment:
I've noticed this line of code in a few places.
`this.destinations.find(dest => dest.id === trip.destinationID)`
You can probably assign this to a variable before line 18 and line 31

`let destination = this.destinations.find(dest => dest.id === trip.destinationID)`

Then on subsequent lines you can just do `destination.propertyName`.

This gives us both a performance improvement, and also makes our code a bit more readable.  You don't have to make these changes just something to think about in the future!